Content.Shared.Database/LogType.cs

```diff
@@ -95,4 +95,6 @@ public enum LogType
     /// </summary>
     ChatRateLimited = 87,
     AtmosTemperatureChanged = 88,
+
+    GameDirector = 100,
```

Comment:
Don't randomly increment logtypes.

Reply:
Ah, that was because I was worrying about it being merged into a fork and having to be incremented every time they merged from upstream.